### ISSUE TITLE
Can't activate the AN 2.1 for Blender 2.8 beta

### ISSUE THREAD

**Issue by uzikisi** [JacquesLucke/animation_nodes#993]
I downloaded the animation nodes and it is there inside the preferences in add ons part but when i try to click the box to activate it the black page is opened
![animation nodes black page](https://user-images.githubusercontent.com/46405580/50728887-901ae300-1139-11e9-841e-03835e12cb97.jpg)

**Comment by OmarSquircleArt** [comment]
Can you open the console and include the error here? Make sure to remove older version of Animation Nodes before installing.

**Comment by uzikisi** [comment]
Hi Omar,

Here is the error message it says multiple versions of animation nodes in
this folder, however i can't see any other animation nodes other than node
wrangler

**Comment by OmarSquircleArt** [comment]
Go to blender's addons directory and delete Animation Nodes manually.

**Comment by OmarSquircleArt** [comment]
As far as I know, on windows, there are two directories where addons are stored. The directory in your blender directory and a user config directory. I am not sure where this is located on Windows. So I recommend you find it and delete Animation Nodes from there.

**Comment by uzikisi** [comment]
Ok, i will do as you recommended, by the way can you tell me exactly which
AN i should download for Blender 2.8 beta in Windows, there are couple, for
example one is master one is ending with 35 one is ending with py37 etc..

İ really thank you a lot, by the way

6 Oca 2019 Paz 23:03 tarihinde Omar Ahmad <notifications@github.com> şunu
yazdı:

**Comment by OmarSquircleArt** [comment]
@uzikisi You should download this one:
https://github.com/JacquesLucke/animation_nodes/releases/tag/v2.1_blender2.8_testbuild1

**Comment by uzikisi** [comment]
Hi Omar,

I erased everything about blender (blender 2.79 and animation nodes 2.0)
then installed Blender 2.8 and animation nodes
animation_nodes_v2.1_blender2.80_windows_py37.zip
<https://github.com/JacquesLucke/animation_nodes/releases/download/v2.1_blender2.8_testbuild1/animation_nodes_v2.1_blender2.80_windows_py37.zip>
but no chance still gives error, cn you help me?  I don't know what to do
regards
<https://github.com/JacquesLucke/animation_nodes/releases/download/v2.1_blender2.8_testbuild1/animation_nodes_v2.1_blender2.80_windows_py37.zip>

**Comment by OmarSquircleArt** [comment]
Can you open the console and tell me what the error is?

**Comment by uzikisi** [comment]
omaaaar, thank you my friend just ater i wrote to you, i managed to install
it, yippuuu, thank you my friend

**Comment by OmarSquircleArt** [comment]
That is good to hear.

**Comment by aduishenov** [comment]
Hi everyone. I was trying to install animation nodes for a while now. 

After installing, it shows up in addons list however when I to activate it, it gives the following:
"There are multiple versions of Animation nodes installed..."
I tried: 
- uninstalling it : didn't work
- downloaded and installed a fresh blender 2.8 from scratch : error persists

After searching for possible solutions, found some recommendation which worked for some people, but not for me. 

But, I found that the addon's data files were not in the primary folder but in this:

**C:\Users\admin1\AppData\Roaming\Blender Foundation\Blender\2.80\scripts\addons**

Hope it will help some of the folks out there.
